charts/camunda-platform/values.yaml

```diff
@@ -1231,7 +1238,7 @@ identity:
     # Image.repository defines which image repository to use
     repository: camunda/identity
     # Image.tag can be set to overwrite the global tag, which should be used in that chart
-    tag:
+    tag: "SNAPSHOT"
```

Comment:
this is the only part that should be removed and added to the caller workflow till the 8.3 release.

Reply:
@aabouzaid @Ben-Sheppard will 8.3 be available next week? Just thinking if it makes sense to wait for the stable 8.3 version if it's available next week.

Reply:
8.3.0 Identity is available now (but not the other components) :)

Reply:
Correct, the releases are planned to be done the end of this week but the official announcements are next week